File: wesignmanagement-client-DEV/src/app/models/results/Reports/usageByCompanyReport.ts
export class usageByCompanyReport {
    public CompanyName: string;
    public GroupName: string;
    public SentDocumentsCount: number;
    public SignedDocumentsCount: number;
    public DeclinedDocumentsCount: number;
    public CanceledDocumentsCount: number;
}

export class usageByCompanyReports {
    public usageByCompaniesReports: usageByCompanyReport[];
}

File: user-backend/MongoIntegratorService/Common/Models/ManagementReports/UsageByCompanyReport.cs
namespace HistoryIntegratorService.Common.Models.ManagementReports
{
    public class UsageByCompanyReport
    {
        public string CompanyName { get; set; }
        public Guid GroupId { get; set; }
        public string GroupName { get; set; }
        public int SentDocumentsCount { get; set; }
        public int SignedDocumentsCount { get; set; }
        public int DeclinedDocumentsCount { get; set; }
        public int CanceledDocumentsCount { get; set; }
    }
}
